src/WebJobs.Extensions.DurableTask/EtwEventSource.cs

```diff
@@ -484,6 +484,36 @@ private EtwEventSource()
             this.WriteEvent(230, TaskHub, AppName, SlotName, Resource, Attempt, DelayMs, Details, ExtensionVersion);
         }
 
+        [Event(231, Level = EventLevel.Warning, Version = 1)]
```

Comment:
Same comment about using the "Info" level instead of "Warning", here and below.

Reply:
Fixed.